mlos_bench/mlos_bench/launcher.py

```diff
@@ -9,9 +9,11 @@
 import logging
 import argparse
 
-from typing import List, Dict, Any
+from typing import Optional, List, Dict, Any
```

Comment:
I think this will get resolved with the merge conflicts, but I tried to make this sorted

Reply:
yep. I've made another pass over the code after resolving the conflicts to make sure that mypy/pylint/flake8/pydocstyle are all happy